turbo/src/main/kotlin/dev/hotwire/turbo/session/TurboSession.kt

```diff
@@ -53,6 +54,7 @@ class TurboSession internal constructor(
     internal val httpRepository = TurboHttpRepository(activity.lifecycleScope)
     internal val requestInterceptor = TurboWebViewRequestInterceptor(this)
     internal val fileChooserDelegate = TurboFileChooserDelegate(this)
+    var isRunningInAndroidNavigation = true
```

Comment:
Public property with default `true`. Set in my library to `false` to change behavior `callback` method behavior.